src/NewTools-FontChooser/StFontChooserPresenter.class.st

```diff
@@ -347,13 +347,18 @@ StFontChooserPresenter >> updatePreview [
 
 { #category : 'initialization' }
 StFontChooserPresenter >> updateWithFont: aFont [
+	"Try to update the receiver's listing with the selected aFont. 
+	If we do not find a font family matching aFont (for example not yet loaded fonts), inform and then select the first family in the system.
+	If we do not find a font style, select the first one available as initial style"
 
```

Comment:
I'm not sure I would select the first one. 
Why not just abort the current update of font?

Reply:
I just reproduced the problem.
I do not know where this presenter is used but If I understand correctly, this presenter can only works if a font is selected, right?
In this case, the fix is ok.